gwtmockito/src/main/java/com/google/gwtmockito/GwtMockitoTestRunner.java

```diff
@@ -264,6 +264,7 @@ public GwtMockitoTestRunner(Class<?> unitTestClass) throws InitializationError {
     packages.add("org.jacoco"); // To support JaCoCo code coverage tools
     packages.add("org.hamcrest"); // Since this package is referenced directly from org.junit
     packages.add("org.junit"); // Make sure the ParentRunner can recognize annotations like @Test
+    packages.add("net.bytebuddy"); // To support Mockito 2
```

Comment:
Move up to alphabetize with the other net package please

Reply:
Done